modules/localgov_expired_events/config/schema/localgov_expired_events.schema.yml

```diff
@@ -0,0 +1,16 @@
+localgov_expired_events.settings:
+  type: config_object
+  label: 'Expired Event module settings'
+  mapping:
+    localgov_expired_events:
+      type: mapping
+      mapping:
+        action:
+          type: boolean
```

Comment:
I think an int would make more sense here, as it could be used as an 'off' state. Also true/false for delete or archive seems strange. Also the code seems to be expecting a string for mapping.action.